maint/qmpi_binding.py

```diff
@@ -270,7 +270,7 @@ def generate_code(parseset: Mapping[str, Any], postfix: str, kind_map: Mapping[s
 
 qmpi_h_file.write(""
         "typedef struct {\n"
-        "    void *storage_stack;\n"
```

Comment:
This change isn't reflected in the `qmpi.h` file. I think it's correct, but it needs to be done in both places.

Reply:
I see the correct lines in both the files. Probably, previously it was manually done directly in the qmpi.h file and missed adding in the qmpi.py file.

Reply:
Ok.